src/main/scripts/language-utilities.js

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
Can https://github.com/ISDCF/registries/blob/master/src/main/scripts/language-utilities.js be reused instead?

Reply:
Done, also updated the main build.js for proc with better error trapping.

Reply:
> Can https://github.com/ISDCF/registries/blob/master/src/main/scripts/language-utilities.js be reused instead?

@palemieux instead of the `git submodule` method and the associated hack of grabbing `language-utilities` from deep inside, what are you thoughts on just publishing a proper NPM package for `ISDCF/registries`. Doing so, would allow packages like this one to simply require it in `package.json`. This would allow for:

1. less hacky ways of getting the version of the registry (version numbers were be baked right into package.json as w/ any other package)
2. Helper utilities (like those found in `language-utilities`) can be properly exported from the package instead of the more brittle method used currently.
3. The dependency of `registries-site` on `registries` become much more explicit.

Reply:
Would a revised package be automatically published to NPM on every single merge to `master` at `ISDCF/registries`?